app/user/invite-users/invite-users.component.html

```diff
@@ -0,0 +1,29 @@
+<md-card class="default-card">
+  <h1>{{ 'invitation' | translate}}</h1>
+</md-card>
+<md-card class="default-card">
+  <div>
+    <md-hint class="error-hint">
+      <span *ngIf = "isError">
+        {{'Please remove following emails' | translate}}:
+        <span *ngFor = "let errorItem of errorItems">
+          <div>{{errorItem}}</div>
+        </span>
```

Comment:
That shows me following:
> Please remove following emails:
> [object Object]

So `Please remove following emails` is missing as translation string, and the `errorItem` is no string.
Also a hint like: "emailAdress is already a user in HiPCMS" would be a better hint.